app/src/main/res/values/array.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="verification_methods">
+        <item>Visual verification</item>
+        <item>Rhythm verification</item>
+        <item>Choose when pairing</item>
+    </string-array>
```

Comment:
Why did you create a separate resource for this? Also (if you want to keep it a separate resource) give it a better name and do the same for the dutch translation :wink: